clients/web/src/app.js

```diff
@@ -25,7 +25,7 @@ import 'girder/stylesheets/layout/global.styl';
 import 'girder/stylesheets/layout/layout.styl';
 
 import 'bootstrap/js/modal';
-import 'girder/utilities/jQuery'; // $.girderModal
+import 'girder/utilities/JQuery'; // $.girderModal
```

Comment:
I very much agree with you. Let me add that to the TODO list on the PR, I would like it to be more explicit, and need to research by looking at a recent version of jQuery, see how they expose functions nowadays.